src/components/Wizard/_index.scss

```diff
@@ -16,4 +18,30 @@
   &__navItem {
     margin-left: -$carbon--spacing-06;
   }
+
+  .#{$prefix}--progress--vertical {
+    .#{$prefix}--progress-step,
+    .#{$prefix}--progress-step-button {
+      flex-wrap: nowrap;
+      margin-bottom: $carbon--spacing-03;
+    }
+
+    .#{$prefix}--progress-step svg,
+    .#{$prefix}--progress-step-button svg {
+      margin-left: 0;
+    }
+
+    .#{$prefix}--progress-label {
+      max-width: 80ch;
+      overflow: hidden;
+      display: -webkit-box;
+      -webkit-line-clamp: 3;
+      -webkit-box-orient: vertical;
+    }
+
+    .#{$prefix}--progress-line {
+      display: none;
+      visibility: hidden;
```

Comment:
```suggestion
```

At first glance, I don't think we need both of these properties, but I'd like to understand your rationale? If it's for code comprehension, I recommend adding a comment to prevent it from being included in the output.